test/test_reports/junit/e2e-results.xml

```diff
@@ -1,8 +1,51 @@
 <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Oops on including this file to begin with.  Need to remove it from git history.

Reply:
will add `test_reports` to gitignore